container_api/container.py

```diff
@@ -18,6 +18,27 @@ def for_id(container_id, sort_function=None, additional_capi_params=None):
 def for_front(front_id, metadata=None):
 	return FrontDataSource(front_id, metadata)
 
+def read_container_title(container_id, retries=3):
```

Comment:
Is it likely that `read_container_title` and the method below it `read_container` are likely to be called at the same time? If so I'm slight concerned we're making two calls for the same endpoint but getting different fields. Perhaps both methods could have parameter to accept either `result` (or `data`?) and parse it to get the relevant field  and we have a separate method to make the api call once and return the response?